### PR TITLE
Add nodejs tool cache for linux.

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
Adding toolset for node.js generation. Reduced minimum free space to 17000MB because of swap file from 8 Gb to 4 Gb
## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/validate-disk-space.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 availableSpaceMB=$(df / -hm | sed 1d | awk '{ print $4}')
-minimumFreeSpaceMB=18000
+minimumFreeSpaceMB=17500
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/validate-disk-space.sh:8]: What is the accurate amount of free disk space will we have after your changes?
I a bit worried that we will affect customers

Comment (dmitry-shibanov) [reply images/linux/scripts/installers/validate-disk-space.sh:8]: 1. success image - https://github.visualstudio.com/virtual-environments/_build/results?buildId=68046&view=results
    Available disk space: 17557 MB
2. failed image - https://github.visualstudio.com/virtual-environments/_build/results?buildId=68044&view=results
    Available disk space: 17557 MB
    Not enough disk space on the image (minimum available space: 17800 MB)

**Pull request merged**